### PR TITLE
[Prettier] Autofix

### PR DESCRIPTION
**Pull request by sampi**
@wiredearp @zdlm @sampi

This might just work. 
How do you guys like the code? I'm ok with most of it, all those newlines around arrays and objects is a little strange, but we could get away with it.

There's still a lot of work to do with https://github.com/Tradeshift/eslint-config-tradeshift but that's left for another day.

### CODE REVIEW

File: docs/src/js/dox-api@tradeshift.com/ts.dox.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 /**
  * Namespace object.
  */
-ts.dox = gui.namespace('ts.dox', (function() {
-	/**
+ts.dox = gui.namespace(
+	'ts.dox',
+	(function() {
+		/**
```
Comment (sampi) [inline docs/src/js/dox-api@tradeshift.com/ts.dox.js]: this is very strange

File: docs/src/js/dox-api@tradeshift.com/ts.dox.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 	 * This is decidedly *not* the case upon GitHub pages.
 	 */
-	var islocalhost = location.port === '10114';
+		var islocalhost = location.port === '10114';
```
Comment (sampi) [inline docs/src/js/dox-api@tradeshift.com/ts.dox.js]: this is strange as well...

File: docs/src/js/dox-api@tradeshift.com/ts.dox.js (hunk 1 above)
Comment (zdlm) [inline docs/src/js/dox-api@tradeshift.com/ts.dox.js]: Can we change this format?

File: docs/src/js/dox-api@tradeshift.com/ts.dox.js [hunk 3]
```diff
@@ -44,94 +46,100 @@ ts.dox = gui.namespace('ts.dox', (function() {
...
 	 */
-	function createtab(tab, file, fold, indx, hide) {
-		var label = tab[0];
-		var href = tab[1];
-		var icon = hide ? 'ts-icon-locked' : null;
-		var magicurl = getmagicurl(fold, href);
-		var selected =	href === file || (indx && href === 'index.html');
-		return new Tab(label, icon, magicurl, selected);
-	}
+		function createtab(tab, file, fold, indx, hide) {
```
Comment (zdlm) [inline docs/src/js/dox-api@tradeshift.com/ts.dox.js]: Looks strange

File: docs/src/js/dox-gui@tradeshift.com/module.js [hunk 4]
```diff
@@ -124,20 +128,25 @@
...
+								};
+							})
+						);
 					} else {
-						console.log(times.map(function(m) {
-							return '	' + m.name + ': ' + m.duration;
-						}).join('\n'));
+						console.log(
+							times
+								.map(function(m) {
```
Comment (zdlm) [inline docs/src/js/dox-gui@tradeshift.com/module.js]: Weird

File: docs/src/js/dox-gui@tradeshift.com/spirits/chrome/ts.dox.ChromeSpirit.js [hunk 5]
```diff
@@ -4,8 +4,7 @@
  * @using {gui.Then} Then
  */
 ts.dox.ChromeSpirit = (function using(CSSPlugin, Then) {
-	var
-		BP_TABLET = 600,
+	var BP_TABLET = 600,
 		SIDEBAR_MACRO = 320,
```
Comment (zdlm) [inline docs/src/js/dox-gui@tradeshift.com/spirits/chrome/ts.dox.ChromeSpirit.js]: Should be align

File: src/runtime/js/ts-polyfilla.js [hunk 6]
```diff
@@ -41,48 +41,63 @@
...
+				(c = string.charCodeAt(i++)) > 255
+			) {
+				throw new TypeError(
+					"Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range."
+				);
 			}
 			bitmap = (a << 16) | (b << 8) | c;
-			result += b64.charAt(bitmap >> 18 & 63) + b64.charAt(bitmap >> 12 & 63) + b64.charAt(bitmap >> 6 & 63) + b64.charAt(bitmap & 63);
+			result +=
+				b64.charAt((bitmap >> 18) & 63) +
```
Comment (zdlm) [inline src/runtime/js/ts-polyfilla.js]: Strange

**Comment by sampi** [comment]
So it seems, unless we switch to spaces, most of the multiline jsdoc comments will be messed up.
We also need to fix up the tradeshift eslint rules, to make sure we're catching issues that are real.

**Comment by sampi** [comment]
We could in theory write a script, that looks for `/**` and checks if the next lines up until `*/` have the same indentation, and fixes them...

**Comment by sampi** [comment]
Also, this is interesting: https://github.com/sheerun/prettier-standard

**Comment by wiredearp** [comment]
This actually doesn't look so great in quite a lot of places. I'm sceptical, but let's study it again this Monday before we decide. The bug with the JSDoc comments is strange though, because that didn't happen with the code I tested Prettier on (which also uses tabs). @zdlm We can't change most of this, but that's also the point, that the "the system" decides once and for all so that developers can stop arguing about it. Some of these strange newlines can perhaps be fixed by setting a longer line length, though. Also: The tab-width appears to be 8 spaces, as if the `.editorconfig` stopped working. This could also explain the unusual (and unaligned) appearance.

File: .editorconfig [hunk 7]
```diff
@@ -8,3 +8,8 @@ end_of_line = lf
 charset = UTF-8
 trim_trailing_whitespace = false
 insert_final_newline = true
+
+
+[package.json]
+indent_style = space
```
Comment (wiredearp) [inline .editorconfig]: @dsp Why spaces? If Prettier uses tabs, and I hope it still does, this will confuse the GitHub code viewer.

**Pull request merged**